.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## What
+<!-- Description of the change being made -->
+
+## Why
+<!-- What are the reasons behind this change being made? -->
+
+## Visual Changes
+<!-- If the change results in visual changes, show a before and after -->
+
+## Pages to check
+
+Locally, as currently no preview app.
+
+* http://service-manual-frontend.dev.gov.uk/service-manual/
```

Comment:
Service Manual Frontend now has review apps, so we can link to these? Not sure how feasible it is to update each link with the PR number for each review app though..

Reply:
I've borrowed from the Publishing Components template, which has the URL commented out and a `[PUT-PR-NUMBER-HERE]` comment in the URL - not automatic, but I think better than leaving it blank.